client/src/components/Header.js

```diff
@@ -56,6 +57,35 @@ function Header() {
     setAnchorEl(null);
   };
 
+  const logout = () => {
+    let status
+    fetch("/logout", {
+      method: "POST",
+      credentials: "include"
+    })
+    .then(res => {
+        status = res.status
+
+        if (status < 500) return res.json();
+        else throw Error("Server error");
+    })
+    .then(res=> {
+        if (status === 200) {
+          history.push("/login")
+          alert(res.response)
+        }
+        else if (status === 500) {
+          alert(res.response)
```

Comment:
Remember to remove the alerts if they were for testing things out.

Reply:
It should be removed after alert context branch is merged